Userland/Applications/Dictionary/Dictionary.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2021, Robin Allen <r@foon.uk>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/MappedFile.h>
+#include <AK/String.h>
+#include <AK/StringView.h>
+
+namespace Dictionary {
+
+class Dictionary {
+public:
+    Dictionary(char const* filename);
+
+    String definition_of(int word_index) const;
+
+    StringView word_at_index(int word_index) const;
+
+    int count_words() const { return m_num_words; }
```

Comment:
The name here suggests that `count_words()` will be performing some work to count how many words there are in the dictionary. Since it's not doing that, I'd rename it to `word_count()` or similar.

Reply:
Done.